ReaderClass/Reader.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+
+namespace ReaderClass
+{
+    public class Reader : IComparable<Reader>
+    {
+        public string Name { get; set; }
+        public string Surname { get; set; }
+
+        public DateTime IssueDate;
+        public readonly int LTicketNumber;
+        public int Period;
+        public readonly DateTime ReturnDate;
+        public decimal Pledge;
+
+        public Reader(string Name, string Surname, string IssueDate, int LTicketNumber, int Period, decimal Pledge)
+        {
+            this.Name = Name;
+            this.Surname = Surname;
+            this.IssueDate = DateTime.Parse(IssueDate);
+            this.LTicketNumber = LTicketNumber;
+            this.Period = Period;
+            ReturnDate = DateTime.Parse(IssueDate).AddDays(Period);
+            this.Pledge = Pledge;
+        }
```

Comment:
Стиль! параметры методов принято называть с маленькой буквы.
```suggestion
        public Reader(string name, string surname, string issueDate, int lTicketNumber, int period, decimal pledge)
        {
            Name = name;
            Surname = surname;
            IssueDate = DateTime.Parse(issueDate);
            LTicketNumber = lTicketNumber;
            Period = period;
            ReturnDate = DateTime.Parse(IssueDate).AddDays(Period);
            Pledge = pledge;
        }
```